ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2017 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+  jcr:primaryType="nt:unstructured"
+  jcr:title="Adaptive Form Button"
+  sling:resourceType="cq/gui/components/authoring/dialog"
+  extraClientlibs="[core.forms.components.button.v1.editor]"
+  trackingFeature="core-components:adaptiveform-button:v1">
+  <content
+    granite:class="cmp-button__editdialog"
+    jcr:primaryType="nt:unstructured"
+    sling:resourceType="granite/ui/components/coral/foundation/container">
+    <items jcr:primaryType="nt:unstructured">
+      <tabs
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/tabs"
+        maximized="{Boolean}true">
+        <items 
+          jcr:primaryType="nt:unstructured"
+          sling:hideChildren="[validation]">
+            <basic
+              jcr:primaryType="nt:unstructured"
+              jcr:title="Basic"
+              sling:resourceType="granite/ui/components/coral/foundation/container">
+              <items 
+                jcr:primaryType="nt:unstructured"
+                sling:hideChildren="[hideTitle,placeholder]">
+                <value
+                  jcr:primaryType="nt:unstructured"
+                  sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                  fieldLabel="Value"
+                  fieldDescription="Value of button"
+                  granite:hide="{Boolean}true"
+                  name="./value"/>
+                <default
+                  jcr:primaryType="nt:unstructured"
+                  sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                  fieldLabel="Default"
+                  fieldDescription="Default value of button"
+                  granite:hide="{Boolean}true"
+                  name="./default"/>
```

Comment:
let's not capture default

Reply:
@vdua  it is hidden, it won't display on ui. I will remove it if not required at all.